README.md

```diff
@@ -2,10 +2,32 @@
 
 # App Scaffold
 
-_Note: the App Scaffold currently depends on zat v1.35.12 or greater._
-
 ## Description
-This repo contains a scaffold to help developers build Iframe apps for Zendesk products.
+This repo contains a scaffold to help developers build [apps for Zendesk](https://developer.zendesk.com/apps/docs/apps-v2/getting_started) products. The App Scaffold includes many features to help you maintain and scale your app. Some of the features provided by the App Scaffold are listed below, however you don't need prior experience in any of these to be able to use the scaffold successfully.
+
+- [ES6 (ES2015)](https://babeljs.io/docs/learn-es2015/)
+
+_ECMAScript 6, also known as ECMAScript 2015, is the latest version of the ECMAScript standard. ES6 is a significant update to the language, and the first update to the language since ES5 was standardized in 2009. Implementation of these features in major JavaScript engines is [underway now](http://kangax.github.io/compat-table/es6/)._
+
+- [Handlebars](http://handlebarsjs.com/) templates
+
+_Handlebars.js is an extension to the Mustache templating language created by Chris Wanstrath. Handlebars.js and Mustache are both logicless templating languages that keep the view and the code separated like we all know they should be._
+
+- [SASS](http://sass-lang.com/) stylesheets
+
+_Sass is an extension of CSS that adds power and elegance to the basic language. It allows you to use variables, nested rules, mixins, inline imports, and more, all with a fully CSS-compatible syntax._
+
+- [Webpack](https://webpack.github.io/) module bundler
+
+_Webpack is a module bundler. It packs CommonJs/AMD modules i. e. for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand._
```

Comment:
this is jargon, maybe it can be a bit more accessible? is this useful?

> Webpack compiles web browser applications. It allows splitting your source code into modules and re-use them with `require` and `import` statements. It also allows splitting your compiled project into separate files that are loaded on demand.